lib/engine/game/g_18_carolinas/step/track.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/track'
+
+module Engine
+  module Game
+    module G18Carolinas
+      module Step
+        class Track < Engine::Step::Track
+          LAY_ACTIONS = %w[lay_tile pass].freeze
+          CONVERT_ACTIONS = %w[run_routes pass].freeze
+          ALL_ACTIONS = %w[lay_tile run_routes pass].freeze
+
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] if entity.company? || !can_lay_tile?(entity) && @mode == :new_track
+            return ALL_ACTIONS if @game.loading && @game.phase.available?('5')
```

Comment:
game.loading? this doesn't seem right

Reply:
It's needed because of the modality. When loading mode is set to track laying mode, so when it encounters a run_routes it fails.